### PR TITLE
Add `itertools.pluck_attr`.

### PR DESCRIPTION
**Pull request by jaheba**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Please tag this pr with at least one of these labels to make our release process faster:** BREAKING, new feature, bug fix, other change, dev setup

### CODE REVIEW

File: src/gluonts/itertools.py [hunk 1]
```diff
@@ -371,3 +372,16 @@ def inverse(dct: Dict[K, V]) -> Dict[V, K]:
     Inverse a dictionary; keys become values and values become keys.
     """
     return {value: key for key, value in dct.items()}
+
+
+_no_default = object()
+
+
+@curry
```
Comment (lostella) [inline src/gluonts/itertools.py:380]: I love curry

File: src/gluonts/itertools.py [hunk 2]
```diff
@@ -371,3 +372,16 @@ def inverse(dct: Dict[K, V]) -> Dict[V, K]:
...
+def pluck_attr(seq, name, default=_no_default):
```
Comment (lostella) [inline src/gluonts/itertools.py:381]: Where does "pluck" come from?

**Review by lostella — APPROVED** [review]

File: src/gluonts/itertools.py (hunk 1 above)
Comment (jaheba) [reply src/gluonts/itertools.py:380]: Such tasty!

Comment (lostella) [reply src/gluonts/itertools.py:380]: Much spice

File: src/gluonts/itertools.py (hunk 2 above)
Comment (jaheba) [reply src/gluonts/itertools.py:381]: Not sure, but it's a common term used. Normally it's for dictionaries.

For example: https://toolz.readthedocs.io/en/latest/api.html#toolz.itertoolz.pluck

**Pull request merged**